tools/clang/test/HLSLFileCheck/hlsl/resource_binding/resource_cleanup.hlsl

```diff
@@ -3,6 +3,7 @@
 // Make sure only one cbuffer is emitted for the final
 // dxil.
 
+// CHECK-LABEL: ; Buffer Definitions:
```

Comment:
Why this change? Seems unrelated to the PR.

Can we get a test to verify the new/expected behavior?

Reply:
This randomly matched "cb1" in the shader hash printed in disassembly causing a false-positive test failure because has `CHECK-NOT: cb1` to make sure there is only one constant buffer in the table.  This label makes sure it skips the hash and starts from the buffer definitions when making sure "cb1" isn't present.

Reply:
Regarding a test to verify new/expected behavior, if you're referring to the version being in the IR, that's going to be a bit hard to test, since the version will depend on build settings and whatnot.  I'm not sure how to test that reliably.